apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html

```diff
@@ -33,118 +42,82 @@
-                    <a
-                        class="nav-link cp"
-                        [ngClass]="{ 'active': smsTabActive }"
-                        (click)="emailTabActive = false; downloadTabActive = false; smsTabActive = true"
-                        >{{ commonLocaleData?.app_send_sms }}</a
-                    >
-                </li>
-            </ul>
-            <div class="pd-2 clearfix">
-                <div [hidden]="!emailTabActive" class="mr-b2">
-                    <textarea
-                        class="form-control"
-                        #emailList
-                        [placeholder]="localeData?.enter_email_addresses"
-                    ></textarea>
-                </div>
-                <div [hidden]="!smsTabActive" class="mr-b2">
-                    <textarea class="form-control" #smsList [placeholder]="localeData?.enter_mobile_no"></textarea>
-                </div>
-                <ng-container *ngIf="currentVoucherFilter === 'invoices'">
-                    <div class="clearfix" [hidden]="smsTabActive">
-                        <h3 class="font-15 mr-b05">{{ localeData?.invoice_copy }}</h3>
-                        <hr />
-                        <div class="checkbox" *ngIf="voucherApiVersion === 2">
-                            <label
-                                ><input
-                                    type="checkbox"
-                                    value="Original"
-                                    checked
-                                    (change)="onSelectInvoiceCopy($event)"
-                                />{{ localeData?.original }}</label
-                            >
-                        </div>
-                        <div class="checkbox">
-                            <label
-                                ><input type="checkbox" value="Transport" (change)="onSelectInvoiceCopy($event)" />{{
-                                    localeData?.transport
-                                }}</label
-                            >
+        <mat-menu #showMoreOptions="matMenu">
+            <div class="more-options" (click)="$event.stopPropagation()">
+                <mat-tab-group 
+                    mat-stretch-tabs="false" 
+                    mat-align-tabs="start"
+                    [selectedIndex]="selectedTabIndex"
+                    (selectedIndexChange)="tabChanged($event)"
+                >
+                    <mat-tab [label]="commonLocaleData?.app_send_email">
+                        <div class="pd-2">
+                            <textarea
+                                matInput
+                                class="form-control"
+                                #emailList
+                                [placeholder]="localeData?.enter_email_addresses"
+                            ></textarea>
+    
+                            <ng-template [ngIf]="currentVoucherFilter === 'invoices' && voucherApiVersion === 2">
```

Comment:
```suggestion
                            <ng-template [ngIf]="currentVoucherFilter === 'invoices' && voucherApiVersion === 2">
```
Please use enum here in over all file